cves/kernel/CVE-2021-3178.yml

```diff
@@ -55,7 +55,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  Exporting a subdirectory of a filesystem allowed unauthorized actors to access
+  other parts of the system using a command called READDIRPLUS.
```

Comment:
I think a small explanation of what `READDIRPLUS` would be helpful. After doing some research, my simple definition would be 

> A method that returns a pointer to a structure that represents a directory entry and all of its attributes at that current directory level

I got this from https://www.pdl.cmu.edu/posix/docs/posix_io_readdir+.pdf